### PR TITLE
Add neptune-fastai

### PR DESCRIPTION
**Pull request by Raalsky**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Raalsky** [comment]
@aniezurawski @HubertJaworski @pitercl @PiotrJander @shnela @pkasprzyk can you all confirm you are willing to be maintainers for this recipe?

File: recipes/neptune-fastai/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+channel_sources:
+  - fastai
```
[the remark this answers is not in the document]
Comment (Raalsky) [reply recipes/neptune-fastai/conda_build_config.yaml:2]: Hi @dopplershift Yeah, we tried :wink:. A few hours before your comment we found info about troubles with maintaining `fastai` package through the `conda-forge` channel (for ex. here: https://twitter.com/jeremyphoward/status/1178351261608861701?lang=en). So I'm not sure we're in ability to maintain our package inside the `coda-forge`. Have you some advices? :wink:

[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/neptune-fastai/conda_build_config.yaml:2]: I think we need more specifics on what problem you're having. conda-forge is inherently an open, community-driven project so we're happy to have people jump in and drive things in a direction--provided it fits with some of our other requirements.

If you're having trouble with how a particular package is being built, we welcome issues regarding that as such on the relevant feedstock (e.g. [torchvision-feedstock](https://github.com/conda-forge/torchvision-feedstock)). If you want to have a more general discussion about some things, I'd recommend our [Gitter Channel](https://gitter.im/conda-forge/conda-forge.github.io).

[the remark this answers is not in the document]
Comment (Raalsky) [reply recipes/neptune-fastai/conda_build_config.yaml:2]: We will reach out to fastai community about the future plans around `conda-forge`. Closing here :wink:

**Pull request closed without merging**